src/app/pages/creator-fest/creator-fest.js

```diff
@@ -8,6 +7,8 @@ import HomeContent from './home-content/home-content';
 import './creator-fest.css';
 
 function PageContent({data, navLinks}) {
+    // Gives reactivity on window.location.pathname
+    // eslint-disable-next-line no-unused-vars
     const location = useLocation();
     const linkEntry = navLinks
```

Comment:
do you have to actually assign to a unused variable? maybe you could just add `useLocation()`